### PR TITLE
Mint discoverability

### PR DESCRIPTION
**Pull request by benthecarman**
for https://github.com/MutinyWallet/mutiny-node/pull/1060

works with my events i created, I don't think this will handle well if information is missing or if there is a ton of people recommending 

also need to do i18n and stuff too

image

### CODE REVIEW

**Comment by futurepaul** [comment]
I rebased and spruced it up a little aesthetically. right now I'm not seeing any recommends and it doesn't seem like my recommend shows up? but it's a super slick ux for adding!

**Comment by benthecarman** [comment]
>  I'm not seeing any recommends and it doesn't seem like my recommend shows up? but it's a super slick ux for adding!

i don't see any created my your npub

**Comment by futurepaul** [comment]
sweet idk if my other wallet was in a bad state nsec-wise, but this worked:

Screenshot 2024-03-25 at 4 27 34 PM

question: is there a good way to check if we've already recommended a fedimint so we can hide the recommend button?

**Comment by benthecarman** [comment]
> question: is there a good way to check if we've already recommended a fedimint so we can hide the recommend button?

hmm, not at the moment, could add some stuff for that. Maybe for now just save in local storage

File: src/routes/settings/ManageFederations.tsx [hunk 1]
```diff
@@ -107,50 +143,229 @@ function AddFederationForm(props: { refetch?: RefetchType }) {
...
+                const myRecommend = thisFederation.recommendations.find(
+                    (r) => r.npub === myHexpub
+                );
+                return myRecommend;
+            }
+        } catch (e) {
+            console.error(e);
+            return undefined;
+        }
+    });
```
Comment (benthecarman) [inline src/routes/settings/ManageFederations.tsx:334]: I'll add a function to mutiny-node that does this more efficiently, but shouldn't be blocking

**Comment by benthecarman** [comment]
can't approve but lgtm

**Comment by futurepaul** [comment]
updated to use https://github.com/MutinyWallet/mutiny-node/pull/1086

**Comment by TonyGiorgio** [comment]
Build is failing, can't test atm

**Comment by TonyGiorgio** [comment]
Screenshot 2024-03-26 at 7 31 27 PM

I'm running on mainnet, this shouldn't be happening.

**Comment by TonyGiorgio** [comment]
This should be fixed with the next RC release. 

In the meantime, can we remove the invite code here? Kind of unnecessary, and when there's 4 or more, it takes up a lot of room and is even more unnecessary.

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**